src/main/java/com/limechain/runtime/Runtime.java

```diff
@@ -25,9 +27,9 @@ public class Runtime {
     private final int heapPages;
     private final FreeingBumpHeapAllocator allocator;
 
-    public Runtime(Module module, int heapPages) {
+    public Runtime(Module module, ImportObject.MemoryImport memoryImport, int heapPages) {
```

Comment:
I couldn't find a more elegant way to pass this as we need it during the `module.instantiate` method. The responsibility to deal with wasm binary stays in the `RuntimeBuilder`  class.